Consolidation of Mautic Form Types (#291)
mautic/developer-documentation-new: docs/webhooks/events/email_on_send.rst

      - Custom Mautic Theme used to style the Preview page or customize Form fields. See :ref:`themes/forms:Customizing forms`.
    * - ``formType``
      - string

>       - **Deprecated.** Previously indicated if a form was ``standalone`` or ``campaign``. As of Mautic 7.1, all forms function identically and this property will be removed in Mautic 8.0.

Reviewer: Let's address the Vale warnings here.

```suggestion
      - **Deprecated.** Previously indicated whether a Form was a Standalone or a Campaign. As of Mautic 7.1, all Forms function identically. Mautic 8.0 schedules the removal of this property.
```

>       - **Deprecated.** Previously indicated whether a Form was a Standalone or a Campaign. As of Mautic 7.1, all Forms function identically. Mautic 8.0 schedules the removal of this property.

Reviewer: @mautic/core-team @RCheesley a question here. Do we need to mention Mautic 8.0 here?

Reviewer 2: It won't hurt but we have different docs like https://github.com/mautic/mautic/blob/7.x/UPGRADE-7.0.md to document breaking changes. So i guess we should start one for M8.

Reviewer 3: This will need to be based on the 7.1 branch indeed - we should add a warning in 7.1 and then in the 8.0 branch it would be removed.
